pkg/log/file_test.go

```diff
@@ -0,0 +1,40 @@
+package log
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func (w *FileLogWriter) WriteLine(line string) error {
+	n, err := w.mw.Write([]byte(line))
+	if err != nil {
+		return err
+	}
+	w.docheck(n)
+	return nil
+}
+
+func TestLogFile(t *testing.T) {
+
+	Convey("When logging to file", t, func() {
```

Comment:
Please remove the test log file after all the test are complete.

Reply:
ok.. log file is removed at the end of the tests.

Reply:
me doing code reviews 
![image](https://user-images.githubusercontent.com/618863/33174491-92719af4-d058-11e7-8b38-3129782f1bc6.png)